bucket4j-spring-boot-starter/src/main/java/com/giffing/bucket4j/spring/boot/starter/config/filter/Bucket4JBaseConfiguration.java

```diff
@@ -36,6 +36,21 @@ public abstract class Bucket4JBaseConfiguration<R, P> implements CacheUpdateList
 
     private final Map<String, ExecutePredicate<R>> executePredicates;
 
+    private final UrlMapper urlMapper;
+
+    protected Bucket4JBaseConfiguration(
+            RateLimitService rateLimitService,
+            CacheManager<String, Bucket4JConfiguration> configCacheManager,
+            List<MetricHandler> metricHandlers,
+            Map<String, ExecutePredicate<R>> executePredicates,
+            @Autowired(required = false) UrlMapper urlMapper) {
```

Comment:
Can we remove the constructor? The default UrlMapper implementation is provided within the Bucket4jUrlConfiguration.